src/libasr/ASR.asdl

```diff
@@ -262,6 +262,7 @@ expr
     | ComplexIm(expr arg, ttype type, expr? value)
     | DictItem(symbol a, expr key, expr? default, ttype type)
     | ListItem(symbol a, expr pos, ttype type, expr? value)
+    | TupleItem(symbol a, expr pos, ttype type, expr? value)
```

Comment:
```suggestion
    | TupleItem(expr a, expr pos, ttype type, expr? value)
```

Reply:
Yes, let me change `symbol` to `expr` for all these nodes in a separate PR.